astropy/utils/iers/iers.py

```diff
@@ -351,8 +371,20 @@ def _check_interpolate_indices(self, indices_orig, indices_clipped, max_input_mj
         class because it has different requirements.
         """
         if np.any(indices_orig != indices_clipped):
-            raise IERSRangeError('(some) times are outside of range covered '
-                                 'by IERS table.')
+            if conf.iers_degraded_accuracy == 'error':
+                msg = ('(some) times are outside of range covered by IERS table. Cannot convert '
+                       'with full accuracy. To allow conversion with degraded accuracy '
+                       'set `astropy.utils.iers.conf.iers_degraded_accuracy` '
```

Comment:
I don't think Sphinx formatting is needed in plain text error message.
```suggestion
                       'set astropy.utils.iers.conf.iers_degraded_accuracy '
```